### DISCUSSION TITLE
just create a furum where people can share ideas or just discuss with each other

### DISCUSSION THREAD

**Discussion by tomatte** [casey/just#2242] [Ideas]
a place where people can discuss and learn from each other the best use cases for just
a place for sharing ideas of command automation

**Comment by casey** [comment]
I think a forum would be a lot of overhead and work, so I think GitHub discussions will have to do for now.

**Reply by tomatte** [reply]
I just realized there's a Discord channel, so no need for a forum and discord is just better :)
